demos/https/iot_demo_https_s3_upload_async.c

```diff
@@ -236,6 +236,8 @@ static void _responseCompleteCallback( void * pPrivData,
                                        IotHttpsReturnCode_t rc,
                                        uint16_t status )
 {
+    ( void ) rc;
```

Comment:
Could rc indicate an error even when status indicates OK?

Reply:
status is HTTP response status. rc is library error. So yes, but this return code is a repeat of the rc from _errorCallback(). Error callback is called first: https://docs.aws.amazon.com/freertos/latest/lib-ref/https/https_design.html#Asynchronous_Callback_Order